frontend/src/Helpers/API.ts

```diff
@@ -1,12 +1,12 @@
 import { showToast } from "../Components/Toaster";
 import { DeathLog, GetDeathLogs } from "./Logging/Death";
 import { FoodLog, GetFoodLogs } from "./Logging/Food";
-import { GetStoryLogs, StoryLog } from "./Logging/StoryLog";
+import { GetStoryLogs, StoryLog, GetStoryMessageLogs, StoryMessageLog} from "./Logging/StoryLog";
 import { Result } from "./Result";
 import { GetSimConfig, SimConfig } from "./SimConfig";
 
 function endpoint(req: string) {
-  return (process.env.REACT_APP_DEV ? "http://localhost:9000/" : window.location) + req;
+  return ("http://localhost:9000/") + req;
```

Comment:
Better not push dev modes :)
We now use env files, you can add a `.env` to `/frontend` and put `REACT_APP_DEV=true`